### PR TITLE
Move language selector out of header to top of page.

### PR DESCRIPTION
**Pull request by dhatch**
Screen Shot 2021-09-14 at 1 39 04 PM

### CODE REVIEW

File: docs/themes/oso-tailwind/assets/js/nav.js [hunk 1]
```diff
@@ -26,13 +26,15 @@ if (sideBarButton) {
...
 }
 
-const langButton = document.getElementById('language-selector-toggle');
-const langsideBar = document.getElementById('language-selector-content');
-if (langButton) {
-  langButton.addEventListener('click', () =>
-    langsideBar.classList.toggle('hidden')
-  );
-}
+window.addEventListener('load', () => {
```
Comment (samscott89) [inline docs/themes/oso-tailwind/assets/js/nav.js:29]: Why is this changed to on load? 

`langButton` needs to be declared for the closing logic below (line 53).

File: docs/config.toml [hunk 2]
```diff
@@ -4,7 +4,7 @@ title = "Oso Documentation"
 theme = ["oso-tailwind", "oso-webpack"]
 uglyurls = true
 
-defaultContentLanguage = "any"
+defaultContentLanguage = "python"
```
Comment (samscott89) [inline docs/config.toml:7]: This is a fantastic idea. Kudos to whoever suggested it

File: docs/themes/oso-tailwind/assets/js/nav.js (hunk 1 above)
Comment (dhatch) [reply docs/themes/oso-tailwind/assets/js/nav.js:29]: Oh might be able to avoid this, it wasn't attaching the event listener but I think that was for a different reason

File: docs/config.toml (hunk 2 above)
Comment (dhatch) [reply docs/config.toml:7]: @gneray

Comment (gkaemmer) [reply docs/config.toml:7]: Does this work? This has the potential to vastly simplify things

Comment (samscott89) [reply docs/config.toml:7]: It does

Comment (samscott89) [reply docs/config.toml:7]: https://docs-preview.oso.dev/v/dhatch-lang-selector-v2

Comment (samscott89) [reply docs/config.toml:7]: Other than the above comment which is breaking things

**Review by samscott89 — APPROVED** [review]
